### DISCUSSION TITLE
Problem with detection of file format in MTEX

### DISCUSSION THREAD

**Discussion by Stefaniacorvo** [mtex-toolbox/mtex#931] [Ask Anything]
Hello,
I use MTEX for grain size calculations, so I usually use file format like .ctf or .cpr that are exported with the EBSD software AztecCrystal.
In recent times I encountered new problems (usually is always ok) relative the file format and I don't know how to solve the problem, could you help me?
![errorMTEX2](https://user-images.githubusercontent.com/76565687/113487872-cb442200-94ba-11eb-927b-3078def462b5.PNG)
![errorMTEX](https://user-images.githubusercontent.com/76565687/113487873-cbdcb880-94ba-11eb-9f09-30381d97d4f3.PNG)

I attach below the error messages that I have, one file that works with MTEX and the two file that give me problems.
Thanks,
Stefania

[FileMTEX.zip](https://github.com/mtex-toolbox/mtex/files/6253216/FileMTEX.zip)

**Comment by ralfHielscher** [comment]
I had a look into your ctf file and found your line 20 quite strange. It is
```
10.196;5.979;9.507	-9.877;-9.877;-9.877	Olivine	1	1			Am. Mineral. [AMMIAY], (1997), vol. 82, pages 913-922
```
Note, the first the numbers are the length of the crystal axes a, b, c. The numbers 4 to 6 are he angles alpha, beta, gamma. 
Why are those negative. This makes no sense at all,

Ralf.
